evm/test/AggregatorProxy_test.js

```diff
@@ -1,10 +1,13 @@
-import * as h from './support/helpers'
-import { assertBigNum } from './support/matchers'
+import * as h from '../src/helpers'
+import { assertBigNum } from '../src/matchers'
+
 const personas = h.personas
 const Aggregator = artifacts.require('Aggregator.sol')
 const AggregatorProxy = artifacts.require('AggregatorProxy.sol')
 const Oracle = artifacts.require('Oracle.sol')
 
+before(h.queryEthClientForConstants)
```

Comment:
Doesn't `Aggregator_test.js` depend on this being called too? Seems like it would fail if you scoped it to only run that file. Ideally there'd be some kind of suite wide global `before` hook.

Reply:
yeah, i have this issue solved in an upcoming PR which makes the entire helpers.ts file stateless.

Reply:
Fixed here: https://github.com/smartcontractkit/chainlink/pull/1684